src/coreclr/md/inc/metamodel.h

```diff
@@ -594,6 +594,7 @@ class CMiniMdBase : public IMetaModelCommonRO
 
 
 protected:
+    DAC_ALIGNAS(8)
```

Comment:
cc: @sdmaclea

Reply:
I am not sure why alignment to 8 is needed. I would rather expect DAC_ALIGNAS(IMetaModelCommonRO).

Reply:
In `src/coreclr/md/inc/VerifyLayouts.inc` there's a 8byte alignment check for m_Schema:
```
BEGIN_TYPE(CMiniMdBase, 8)                                           //vtable ptr and first field 8-byte alignment
ALIGN_FIELD(CMiniMdBase, m_Schema, sizeof(CMiniMdSchema), 8)
```

This check started to fail after recent changes to CMiniMdBase and CMiniMdSchema, where new class fields were added. So, I've added explicit 8 byte alignment